app/appshell/app.js

```diff
@@ -210,8 +211,8 @@ app.getRemoteDebuggingPort = function () {
 };
 
 app.getUserDocumentsDirectory = function () {
-    // TODO: implement
-    throw new Error("app.getUserDocumentsDirectory not implemented");
+    var _path = path.resolve(electronApp.getPath("home"), "Documents");
```

Comment:
This might also fail on other languages on OSX. I wonder, if there is a better way to determine the document directory location. Environment variable per platform that we can query? This could help to resolve the translated documents/dokumente folder issue.